tests/README.md

```diff
@@ -0,0 +1,92 @@
+# s2n-tls testing
+s2n-tls has a wide variety of tests covering a wide variety of different functionality. This document serves as a "quick reference", providing an overview of all the testing that s2n-tls does and linking to additional documentation.
+
+## Unit Tests
+_Is the internal functionality of s2n-tls correct?_
+
+These test the correctness of low-level s2n-tls functions. These functions are commonly private. We have unit tests for both the C library and the [rust bindings](../bindings/rust). 
+
+### Sanitizers
+The C unit tests are additionally run with the following sanitizers enabled
+- Valgrind: used to detect memory errors
+- Address Sanitizer (ASAN): used to detect memory errors
+- Thread Sanitizer (TSAN): used to detect race conditions
+
+Further information:
+- [C tests](unit/)
```

Comment:
It's a bit tricky that relative links like this don't resolve right in this PR. Maybe take an action item to double check them all once you've merged?

Reply:
Yup! I already checked this. You can view the rendered markdown (with clickable links) by going
1. File Diff
2. `...` menu
3. View File

At which point you should see the fully rendered markdown file.

Reply:
Oh that's a neat trick! Good to know.